.github/workflows/build_container_pull.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+name: Build container for stage testing
+on: [push]
```

Comment:
Do we really want to build a new container on every push to every branch?

Reply:
changed to pull request